databroker/core.py

```diff
@@ -657,15 +847,17 @@ def _load(self):
                 lookup_resource_for_datum=self._lookup_resource_for_datum,
                 get_datum_pages=self._get_datum_pages,
                 fillers=self.fillers,
-                metadata={'descriptors': descriptors})
+                metadata={'descriptors': descriptors,
+                          'resources': self._resources})
             self._entries[stream_name] = intake.catalog.local.LocalCatalogEntry(
                 name=stream_name,
                 description={},  # TODO
                 driver='databroker.core.BlueskyEventStream',
                 direct_access='forbid',
                 args=args,
-                cache=None,  # ???
-                metadata={'descriptors': descriptors},
+                cache=[self._cache],  # This doesn't seem to do anything
```

Comment:
I think we should revert this change at least until we understand exactly what this does.

Reply:
updated